hbase-client/src/main/java/org/apache/hadoop/hbase/Size.java

```diff
@@ -0,0 +1,157 @@
+/**
+ * Copyright The Apache Software Foundation
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.hadoop.hbase;
+
+import java.math.BigDecimal;
+import java.util.Objects;
+
+import org.apache.hadoop.hbase.classification.InterfaceAudience;
+
+/**
+ * It is used to represent the size with different units.
+ * This class doesn't serve for the precise computation.
+ */
+@InterfaceAudience.Public
```

Comment:
I'm not sure, but does it need to be IA.Public? A class in hbase-client package doesn't mean it must be open to clients, like `RegionLocation`.

Reply:
It's a backport from the master branch actually and it's `@InterfaceAudience.Public` there. Should we change it to `@InterfaceAudience.Private` in branch-1?

Reply:
Maybe we can move it to the hbase-hbtop module and change to `@InterfaceAudience.Private`. Will do that.

Reply:
Done.

Reply:
Can provide an addendum to master & branch-2s. Not a big problem, I think.

Reply:
Actually, the `Size` class is used by other classes in master & branch-2s and I just reused the class in the hbtop module. So we don't need to move the class to anywhere in master & branch-2s. In branch-1, we don't have the class so we needed to port the class to branch-1.